ci-operator/step-registry/opct/test/platform-external-vsphere-compact/opct-test-platform-external-vsphere-compact-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  # https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/40077/pull-ci-openshift-release-master-ci-operator-config/1671870759988891648
+  as: opct-test-platform-external-vsphere-compact
+  # as: opct-test-platform-external-vsphere-compact-workflow
+  steps:
+    env:
+      WORKER_REPLICAS: "0"
+      MASTER_REPLICAS: "3"
+    pre:
+      - chain: upi-vsphere-platform-external-ovn-pre
```

Comment:
@faermanj it seems [this error](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/40077/pull-ci-openshift-release-master-ci-operator-registry/1678430377036746752) points to declare the vars in the step `upi-vsphere-platform-external-ovn-pre`. Is it correct? Are using this chain  to your setup?

In the PR I can see only the new chain `upi-conf-vsphere-platform-none-compact-ovn`, if you want to use this one, you need to define the env there.

Reply:
This was probably copied from the reference, i'll try changing to -compact, thanks!